data/access_token_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2016, German Neuroinformatics Node (G-Node),
+//                     Adrian Stoewer <adrian.stoewer@rz.ifi.lmu.de>
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted under the terms of the BSD License. See
+// LICENSE file in the root of the Project.
+
+package data
+
+import (
+	"github.com/G-Node/gin-auth/util"
+	"testing"
+	"time"
+)
+
+const (
+	accessTokenAlice = "3N7MP7M7"
+	accessTokenBob   = "LJ3W7ZFK" // is expired
+)
+
+func TestListAccessTokens(t *testing.T) {
+	defer failOnPanic(t)
+	initTestDb(t)
+
+	accessTokens := ListAccessTokens()
+	if len(accessTokens) != 2 {
+		t.Error("Exactly to access tokens expected in slice")
+	}
+}
+
+func TestGetAccessToken(t *testing.T) {
+	defer failOnPanic(t)
+	initTestDb(t)
+
+	tok, ok := GetAccessToken(accessTokenAlice)
+	if !ok {
+		t.Error("Access token does not exist")
+	}
+	if tok.AccountUUID != uuidAlice {
+		t.Errorf("AccountUUID was expectd to be '%s'", uuidAlice)
+	}
+
+	_, ok = GetAccessToken("doesNotExist")
+	if ok {
+		t.Error("Access token should not exist")
+	}
+}
+
+func TestClearOldAccessTokens(t *testing.T) {
+	defer failOnPanic(t)
+	initTestDb(t)
+
+	deleted := ClearOldAccessTokens()
+	if deleted != 1 {
+		t.Error("Exactly one access token is supposed to be deleted")
+	}
+
+	_, ok := GetAccessToken(accessTokenBob)
+	if ok {
+		t.Error("Bobs access token should not exist")
+	}
+}
+
+func TestCreateAccessToken(t *testing.T) {
+	initTestDb(t)
+
+	token := util.RandomToken()
+	new := AccessToken{
```

Comment:
`new` is a keyword, but I guess its quite ok to use it as a var name in the simple test.